src/resources/css/style.css

```diff
@@ -3,6 +3,11 @@
  *
  * @package TribeEventsCalendarPro
  */
+
+.tribe_ext_sch_day_view #tribe-events-pg-template {
```

Comment:
This didn't take effect for me with Twenty Seventeen theme, but this CSS did:
`.tribe_ext_sch_day_view .tribe-events-day .tribe-events-loop { max-width: 100%; }`

Does that CSS solve the issue for you too on your theme?

Reply:
On 2017, it's not full width, but, it is the same width as List View (which is fine) and it doesn't get "stuck" on resize. So - I think the change as I presented it is good. 

Your recommended change would _also_ make it full width in 2017, but I'm not sure we need to do that.

Reply:
Gotcha. 👍